micropython/helpers.py

```diff
@@ -124,9 +128,30 @@ def post(server: str, path: str, headers: list, data: bytes, debug: bool = False
     return response
 
 
+def post(server: str, path: str, headers: list, data: bytes, debug: bool = False) -> any:
+    return request("POST", server, path, headers, data, debug)
+
+
+def get(server: str, path: str, headers: list, debug: bool = False) -> any:
+    return request("GET", server, path, headers, debug=debug)
```

Comment:
Why debug = debug here but not in post?

Reply:
because here we are skipping the "data" argument, so we have to say which one "debug" is